package/MDAnalysis/transformations/__init__.py

```diff
@@ -26,34 +26,72 @@
 Trajectory transformations --- :mod:`MDAnalysis.transformations`
 ================================================================
 
-The transformations submodule contains a collection of functions to modify the
-trajectory. Coordinate transformations, such as PBC corrections and molecule fitting
-are often required for some analyses and visualization, and the functions in this
-module allow transformations to be applied on-the-fly.
-These transformation functions can be called by the user for any given
-timestep of the trajectory, added as a workflow using :meth:`add_transformations`
-of the :mod:`~MDAnalysis.coordinates.base` module, or upon Universe creation using
+The transformations submodule contains a collection of function-like classes to
+modify the trajectory.
+Coordinate transformations, such as PBC corrections and molecule fitting
+are often required for some analyses and visualization, and the functions in
+this module allow transformations to be applied on-the-fly.
+
+A typical transformation class looks like this:
+
+.. code-blocks:: python
+
+    class transformation(object):
+        def __init__(self, *args, **kwargs):
+            #  do some things
+            #  save needed args as attributes.
+            self.needed_var = args[0]
+
+        def __call__(self, ts):
+            #  apply changes to the Timestep object
+            return ts
+
+See `MDAnalysis.transformations.translate` for a simple example.
+
+These transformation functions can be called by the user for any given timestep
+of the trajectory, added as a workflow using :meth:`add_transformations`
+of the :mod:`~MDAnalysis.coordinates.base`, or upon Universe creation using
 the keyword argument `transformations`. Note that in the two latter cases, the
-workflow cannot be changed after being defined.
+workflow cannot be changed after being defined. for example:
 
-In addition to the specific arguments that each transformation can take, they also
-contain a wrapped function that takes a `Timestep` object as argument.
-So, a transformation can be roughly defined as follows:
+.. code-block:: python
+
+    u = mda.Universe(GRO, XTC)
+    ts = u.trajectory[0]
+    trans = transformation(args)
+    ts = trans(ts)
+
+    #  or add as a workflow
```

Comment:
I think we only want to promote the workflow usage, else it's just a function they can (forget to) apply